cpp/tests/grouped_rolling/grouped_rolling_test.cpp

```diff
@@ -36,6 +36,85 @@
 using cudf::test::fixed_width_column_wrapper;
 using cudf::size_type;
 using cudf::bitmask_type;
+ 
+const std::string cuda_func{
+  R"***(
+    template <typename OutType, typename InType>
+    __device__ void CUDA_GENERIC_AGGREGATOR(OutType *ret, InType *in_col, cudf::size_type start,
+                                            cudf::size_type count) {
+      OutType val = 0;
+      for (cudf::size_type i = 0; i < count; i++) {
+        val += in_col[start + i];
+      }
+      *ret = val;
+    }
+  )***"};
+
+const std::string ptx_func{
```

Comment:
I am not qualified to review this code without doing a good bit of research first.